apps/style/netsim/style.scss

```diff
@@ -413,11 +410,15 @@ svg {
         color: #d3d3d3;
       }
 
-      .bit_counter {
+      .bit-counter {
         text-align: right;
         color: darkgray;
       }
 
+      .bit-counter.oversized {
+        color: #ff0000;
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.